src/app/OperationalDeviceManager.cpp

```diff
@@ -0,0 +1,181 @@
+/*
```

Comment:
Please do the rename in a separate PR from substantive changes that are big enough that git does not detect the rename.  Otherwise reviewing is a huge pain....

@pan-apple Please look this over and see whether it fits in with the planned evolution of this class and CASE sessions?